modules/apps/change-tracking/change-tracking-test/src/testIntegration/java/com/liferay/change/tracking/internal/portlet/action/test/InviteUsersMVCResourceCommandTest.java

```diff
@@ -206,30 +195,31 @@ private ThemeDisplay _getThemeDisplay() throws Exception {
 		return themeDisplay;
 	}
 
-	private void _inviteUserToPublication() throws Exception {
-		CTCollection ctCollection = _ctCollectionLocalService.addCTCollection(
-			null, TestPropsValues.getCompanyId(), TestPropsValues.getUserId(),
-			0, RandomTestUtil.randomString(), null);
+	private void _testGetInviteUsers(CTCollection ctCollection)
+		throws Exception {
+
+		_serviceContext = ServiceContextTestUtil.getServiceContext(
+			TestPropsValues.getGroupId(), TestPropsValues.getUserId());
 
 		User user = UserTestUtil.addUser();
 
 		MockLiferayResourceRequest mockLiferayResourceRequest =
 			_getMockLiferayResourceRequest(
 				ctCollection.getCtCollectionId(), user.getUserId());
 
-		ServiceContext serviceContext =
-			ServiceContextTestUtil.getServiceContext(
-				TestPropsValues.getGroupId(), TestPropsValues.getUserId());
-
-		serviceContext.setRequest(
+		_serviceContext.setRequest(
 			mockLiferayResourceRequest.getHttpServletRequest());
 
-		ServiceContextThreadLocal.pushServiceContext(serviceContext);
+		ServiceContextThreadLocal.pushServiceContext(_serviceContext);
 
 		_mvcResourceCommand.serveResource(
 			mockLiferayResourceRequest, new MockLiferayResourceResponse());
+
+		ServiceContextThreadLocal.popServiceContext();
```

Comment:
Wrap this in a try-finally block to ensure that we always pop the service context even if unexpected exceptions occur in the resource command.